index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
```

Comment:
Great use of indentation and spacing throughout file! Especially helped with keeping track of tags